en/docs/develop/debugging-mediation.md

```diff
@@ -23,23 +23,34 @@
    
           <img src="{{base_path}}/assets/img/develop/debugger/remove-breakpoint.png" alt="remove breakpoint" width="300">
 
-6. Select the **Run and Debug** extension from the left side panel and then click on the **Run and Debug** button.
+6. Configure a fresh MI server (server without already deployed integrations) to avoid conflicts by following the steps below.
+
+    - Open the command palette in VS Code by entering `Command`+`Shift`+`P` on macOS and `Ctrl`+`Shift`+`P` on Windows.
```

Comment:
Please use an ordered (nested) list for this procedure, as it involves sequential steps.